.github/workflows/examples.yml

```diff
@@ -17,6 +17,7 @@ on:
       - ready_for_review
       - reopened
       - synchronize
+      - labeled
```

Comment:
I'm pretty sure this won't work. The `paths` filter for `pull_request` would still hold.

Reply:
Oh I see. I will take a look at it.